tests/tests_fabric/test_fabric.py

```diff
@@ -1177,12 +1178,20 @@ def test_verify_launch_called():
     with pytest.raises(RuntimeError, match=r"you must call `.launch\(\)`"):
         fabric._validate_launched()
 
+    # Methods
     method_names = ("setup", "setup_module", "setup_dataloaders", "broadcast", "barrier", "all_reduce", "all_gather")
     for method_name in method_names:
         method = getattr(fabric, method_name)
         with pytest.raises(RuntimeError, match=r"you must call `.launch\(\)`"):
             method(Mock())
 
+    # Context managers
+    ctx_manager_names = ("init_module",)
+    for ctx_manager_name in ctx_manager_names:
+        ctx_manager = getattr(fabric, ctx_manager_name)
+        with pytest.raises(RuntimeError, match=r"you must call `.launch\(\)`"), ctx_manager():
+            pass
```

Comment:
what does this do? :thinking:

Reply:
This asserts that the runtime error is raised by the context manager. It is equivalent to

```py
with pytest.raises(...):
    with ctx_manager(...):
        pass
```

Reply:
ok, I was confused with the pass inside, so maybe add a comment for future me?